### DISCUSSION TITLE
Open internal links in current tab and external links in new tab.

### DISCUSSION THREAD

**Discussion by linuxreader** [McShelby/hugo-theme-relearn#854] [Q&A] [answered]
Hello,

How do I edit link behavior? Right now. everything opens in a new tab which is kind of annoying. Is there a way to have links open in the current tab? And maybe have external links open in a new tab?

**Comment by McShelby** [comment]
Internal links are always opened in the same window, external links are by default opened in another tab. This can be configured via `externalLinkTarget` in your `hugo.toml`. See https://mcshelby.github.io/hugo-theme-relearn/search.html?search-by=externalLinkTarget

**Reply by linuxreader** [reply]
Can internal links be opened in the current tab instead of a new tab?

**Reply by McShelby** [reply, marked as answer]
> Can internal links be opened in the current tab instead of a new tab?

They are. The question here is: What does the theme think, _is_ an internal link? Anything that does not start with a protocol specification in the URL.

So if you use `[website setup](https://perfectdarkmode.com/posts/2023/03/building-a-minimalist-website-with-hugo/)` it will be handled as external link. Use relative links instead.

**Comment by linuxreader** [comment]
This helps a ton. Thanks for the clarification!
